index.html

```diff
@@ -1,29 +1,30 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
-    <meta charset="UTF-8" />
-    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Home page</title>
-    <link rel="stylesheet" href="styles.css" />
-  </head>
-  <body>
-    <img src="img/seven-o-clock.svg" alt="Animated clock logo" class="logo" />
-    <section class="head-line">
-      <header class="tool-bar">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+     <link rel="stylesheet" href="styles.css"> 
+     
+</head>
+<body>
+  
+  <section class="head-line">
+    <header class="tool-bar" id="navbar-js">
         <div class="toolbar-dark-compact">
-          <a class="john-doe" href="https://www.w3schools.com">Yasen</a>
-          <nav class="normal-button">
-            <img class="humberger-img" src="img/Union.png" alt="Mobile menu" />
-            <ul class="drop-content">
-              <li><a href="">Portfolio</a></li>
-              <li><a href="">About</a></li>
-              <li><a href="">Contact</a></li>
-              <li>
-                <a href=""><img src="img/envelop.svg" alt="Email" /></a>
-              </li>
-            </ul>
-          </nav>
+            <a class="john-doe" href="https://www.w3schools.com">Yasen</a>
+            <nav class="normal-button" >  
+                <img  id="humberger-icon" class="humberger-img" src="img/Union.png" alt="">
+                <ul class="drop-content">
+                    <li><a href="">Portfolio</a> </li>
+                    <li><a href="">About</a></li>
+                    <li><a href="">Contact</a></li>
+                    <li><a href=""><img src="img/envelop.svg" alt=""></a></li>
+                   </ul>
+            </nav>
```

Comment:
- [ ] Good job implementing the mobile menu functionality 👍. I think it would be better if your mobile menu design matches the Figma design template you chose. 

Currently, the mobile menu text is too bold and the margin between the menus is much. Kindly fix it.

| Figma Design  | Your Design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/63473584/180853778-37135a9d-e1c8-46f1-bba7-07f6b61d845e.png)  | ![image](https://user-images.githubusercontent.com/63473584/180854003-b8f9870a-85fc-41cb-a312-0116bdffae75.png)  |